cartridges/bm_algolia/cartridge/templates/default/algoliabm/dashboard/index.isml

```diff
@@ -149,6 +149,18 @@
                     </td>
                 </tr>
 
+                <iscomment> Algolia_LocalesForIndexing </iscomment>
+                <tr>
+                    <td class="table_detail w s" colspan="1">
+                        <isprint value="${Resource.msg('algolia.label.preference.locales', 'algolia', null)}" encoding="jshtml" />
+                        <i class="fa fa-info-circle dw-nc-text-info info-hover"></i>
+                        <div class="tooltip">${Resource.msg('algolia.label.preference.locales.help', 'algolia', null)}</div>
+                    </td>
+                    <td class="table_detail w e s">
```

Comment:
Do you think that can we use a more user friendly input( multi-select checkbox etc) here? I am not sure if it is worth to do

Reply:
Thanks for the idea. So that's not what we had in mind initially, but it worth thinking about.
For now I'm not sure it would make the UI more friendly because some sites have >100 locales, so the UI would be flooded with checkboxes in that case.
Also, if none are selected, we want to index them all by default, which is a bit counter-intuitive.
But happy to discuss the idea further.